examples/qml_features/qml/pages/CustomBaseClassPage.qml

```diff
@@ -76,4 +77,13 @@ Page {
             }
         }
     }
+
+    BusyIndicator {
+        anchors {
+            right: scrollView.right
+            bottom: scrollView.bottom
+            margins: 15
+        }
+        running: customBaseClass.state == CustomBaseClass.Running
```

Comment:
this should work with triple equals and would be good to test that it does 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality